tgui/packages/tgui/interfaces/Clipboard.js

```diff
@@ -34,10 +38,6 @@ export const Clipboard = (props, context) => {
                 {pen}
               </LabeledList.Item>
             </LabeledList>
-          ) : (integrated_pen ? (
```

Comment:
hnnnghhhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH non modular interface changes will bite you in the ass later.

Reply:
Oh I forgot to undo this, since I did that upstream change.

Reply:
Actually, I'm going to PR that change upstream because I just realized that this was the better way of doing it.